diff --git a/node_modules/quill/dist/quill.js b/node_modules/quill/dist/quill.js
index 811b3d0..8d765ad 100644
--- a/node_modules/quill/dist/quill.js
+++ b/node_modules/quill/dist/quill.js
@@ -4517,7 +4517,8 @@ var Keyboard = function (_Module) {
       }
     });
     _this.addBinding({ key: Keyboard.keys.ENTER, shiftKey: null }, handleEnter);
-    _this.addBinding({ key: Keyboard.keys.ENTER, metaKey: null, ctrlKey: null, altKey: null }, function () {});
+    _this.addBinding({ key: Keyboard.keys.ENTER, shiftKey: null }, handleSpecialEnter);
+    _this.addBinding({ key: Keyboard.keys.ENTER, metaKey: null, ctrlKey: null, altKey: null }, function () { return true });
     if (/Firefox/i.test(navigator.userAgent)) {
       // Need to handle delete and backspace for Firefox in the general case #1171
       _this.addBinding({ key: Keyboard.keys.BACKSPACE }, { collapsed: true }, handleBackspace);
@@ -4939,6 +4940,15 @@ function handleEnter(range, context) {
     if (name === 'link') return;
     _this3.quill.format(name, context.format[name], _quill2.default.sources.USER);
   });
+  return true
+}
+  
+function handleSpecialEnter(range, context) {
+  const text = this.quill.getText(range.index + 1, 1);
+  const isInLast = text === '\n';
+  this.quill.deleteText(range.index + (isInLast ? 1 : 0), 1, _quill2.default.sources.SILENT)
+  this.quill.focus();
+  return true
 }
 
 function makeCodeBlockHandler(indent) {
